patches/minecraft/net/minecraft/world/level/block/BaseRailBlock.java.patch

```diff
@@ -35,30 +35,31 @@
              p_49385_.m_46672_(p_49386_.m_7494_(), this);
           }
  
-@@ -139,6 +_,7 @@
+@@ -139,6 +_,11 @@
        return blockstate.m_61124_(this.m_7978_(), flag1 ? RailShape.EAST_WEST : RailShape.NORTH_SOUTH).m_61124_(f_152149_, Boolean.valueOf(flag));
     }
  
-+   @Deprecated //Forge: Use getRailDirection(IBlockAccess, BlockPos, IBlockState, EntityMinecart) for enhanced ability
++   /**
```

Comment:
This is a regular comment and not a javadoc. This is to avoid conflicts with mapping sets (e.g. parchment) adding their own javadoc.